tests/arrays_test_generated.h

```diff
@@ -140,22 +149,34 @@ FLATBUFFERS_MANUALLY_ALIGNED_STRUCT(8) ArrayStruct FLATBUFFERS_FINAL_CLASS {
   static const flatbuffers::TypeTable *MiniReflectTypeTable() {
     return ArrayStructTypeTable();
   }
-  ArrayStruct() {
-    memset(static_cast<void *>(this), 0, sizeof(ArrayStruct));
+  ArrayStruct()
+      : a_(0),
+        b_(),
+        c_(0),
+        padding0__(0),
+        padding1__(0),
+        padding2__(0),
+        d_(),
+        e_(0),
+        padding3__(0),
+        f_()
+  {
+    (void)padding0__;
+    (void)padding1__;
+    (void)padding2__;
```

Comment:
unfortunately, yes
otherwise, compiler (clang 10.0) says something like
```
In file included from flatbuffers/tests/test.cpp:39:
flatbuffers/build/tests/arrays_test_generated.h:71:10: error: private field 'padding0__' is not used [-Werror,-Wunused-private-field]
  int8_t padding0__;  int32_t padding1__;
```